test/suites/ACID.m

```diff
@@ -2458,6 +2459,31 @@
   set(h, 'orientation', 'horizontal');
 end
 % =========================================================================
+function [stat] = pruneOutsideBox()
+    % test function 'pruneOutsideBox' that is part of 'cleanfigure'
+    stat.description = ['Test function \texttt{pruneOutsideBox()} ', ...
+                        'of \texttt{cleanfigure()}.'];
+    stat.issues      = 226;
+
+    % generate 16 deterministic values
+    data = magic(4);
+    data = data(:);
+
+    % create plots
+    subplot(2,1,1);
+    plot(data,'x-');
+    xlim([10 12]);      % only show 3 values
+    title('regular plot');
+
+    subplot(2,1,2);
+    stairs(data,'x-');
+    xlim([10 12]);      % only show 2 steps
+    title('stairs plot');
+
+    % force application of 'cleanfigure()'; don't rely on testsuite
+    cleanfigure();
```

Comment:
@egeerardyn @PeterPablo I interpreted the test in a wrong way. Now I get what we want to test. Still, I think the following testing case shows more clearly the states of the figure, i.e. before and after cleanfigure:

![capture](https://cloud.githubusercontent.com/assets/3731173/5958333/697abe3c-a7bb-11e4-9f37-dcb02550cbbc.PNG)

You can clearly **see** the effects of cleanfigure. The figure is before any fixes, thus the blue line also disappears and stairs are not captured in R2014b. I reckon this one 


```matlab
% Setup
subplot(311)
title setup
hold on
plot(1:10,10:-1:1,'-r*',1:15,repmat(9,1,15),'-g*',[5.5,5.5],[1,9],'-b*')
stairs(1:10,'-m*')
plot([2,8.5,8.5,2,2],[2,2,7.5,7.5,2],'--k')
legend('cross with points','no cross','cross no points','stairs','zoom area')

% Last comes before simple zoomin due to cleanfigure
subplot(313)
title 'zoom in, cleanfigure, zoom out'
hold on
plot(1:10,10:-1:1,'-r*',1:10,repmat(9,1,10),'-g*',[5.5,5.5],[1,9],'-b*')
stairs(1:10,'-m*')
xlim([2, 8.5]), ylim([2,7.5])
cleanfigure()
plot([2,8.5,8.5,2,2],[2,2,7.5,7.5,2],'--k')
xlim([0, 15]), ylim([0,10])

% Simple zoom in
subplot(312)
title 'zoom in'
hold on
plot(1:10,10:-1:1,'-r*',1:10,repmat(9,1,10),'-g*',[5.5,5.5],[1,9],'-b*')
stairs(1:10,'-m*')
xlim([2, 8.5]), ylim([2,7.5])
```